<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElogWindow</class>
 <widget class="QWidget" name="ElogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic logbook</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/nicos-gui.qrc">
    <normaloff>:/paste</normaloff>:/paste</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Add a &amp;Remark (current experiment status):</string>
         </property>
         <property name="buddy">
          <cstring>remarkText</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remarkText"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addRemark">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="creoleLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:7.5pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Add &amp;amp;free text, Creole formatted (&lt;/span&gt;&lt;a href=&quot;help&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#0057ae;&quot;&gt;help&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="buddy">
          <cstring>freeFormText</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="freeFormText"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addFreeForm">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Attach a file:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;File:</string>
           </property>
           <property name="buddy">
            <cstring>fileName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fileName"/>
         </item>
         <item>
          <widget class="QPushButton" name="fileSelect">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../resources/nicos-gui.qrc">
             <normaloff>:/open</normaloff>:/open</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Re&amp;name to:</string>
           </property>
           <property name="buddy">
            <cstring>fileRename</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fileRename"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Description:</string>
           </property>
           <property name="buddy">
            <cstring>fileRename</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fileDesc"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addFile">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="refreshLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:7.5pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Log preview:  &lt;/span&gt;&lt;a href=&quot;refresh&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#0057ae;&quot;&gt;refresh&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;  &lt;/span&gt;&lt;a href=&quot;back&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;back&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;  &lt;/span&gt;&lt;a href=&quot;forward&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;forward&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWebView" name="preview">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/print</normaloff>:/print</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Close window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUnzoom">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/viewfull</normaloff>:/viewfull</iconset>
   </property>
   <property name="text">
    <string>&amp;Unzoom</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as PDF...</string>
   </property>
   <property name="toolTip">
    <string>Save plot as PDF</string>
   </property>
  </action>
  <action name="actionLogScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Logarithmic Y scale</string>
   </property>
   <property name="iconText">
    <string>Log Scale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCloseView">
   <property name="text">
    <string>Close view</string>
   </property>
   <property name="iconText">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close plot for this view</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>L&amp;egend</string>
   </property>
  </action>
  <action name="actionDeleteView">
   <property name="text">
    <string>Delete view</string>
   </property>
   <property name="iconText">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete view</string>
   </property>
  </action>
  <action name="actionResetView">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../../resources/nicos-gui.qrc">
     <normaloff>:/new</normaloff>:/new</iconset>
   </property>
   <property name="text">
    <string>New view</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>remarkText</tabstop>
  <tabstop>addRemark</tabstop>
  <tabstop>fileName</tabstop>
  <tabstop>fileSelect</tabstop>
  <tabstop>fileRename</tabstop>
  <tabstop>addFile</tabstop>
  <tabstop>freeFormText</tabstop>
  <tabstop>addFreeForm</tabstop>
  <tabstop>preview</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>ElogWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
